run/idp-sql/cloud-run-button-script.sh

```diff
@@ -0,0 +1,64 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+export _SECRET_NAME="vote-sql-secrets"
+export _SERVICE_ACCOUNT="vote-indentity"
+
+gcloud config set project $GOOGLE_CLOUD_PROJECT
+
+# Enable Cloud SQl and Secret Manager APIs
+gcloud services enable sqladmin.googleapis.com secretmanager.googleapis.com
+
+gcloud sql instances describe ${CLOUD_SQL_INSTANCE_NAME}
+if [ $? -eq 1 ]; then
+  echo "Create Cloud SQL instance with postgreSQL database ..."
+  gcloud sql instances create ${CLOUD_SQL_INSTANCE_NAME} \
+      --database-version=POSTGRES_12 \
+      --region=${GOOGLE_CLOUD_REGION} \
+      --cpu=2 \
+      --memory=7680MB \
+      --root-password=${DB_PASSWORD}
+fi
+
+sed -i "s/PROJECT_ID/$GOOGLE_CLOUD_PROJECT/" postgres-secrets.json
+sed -i "s/REGION/$GOOGLE_CLOUD_REGION/" postgres-secrets.json
+sed -i "s/PASSWORD_SECRET/$DB_PASSWORD/" postgres-secrets.json
+sed -i "s/INSTANCE/$CLOUD_SQL_INSTANCE_NAME/" postgres-secrets.json
+
+gcloud secrets describe ${_SECRET_NAME}
+if [ $? -eq 1 ]; then
+  echo "Creating secret ..."
+  gcloud secrets create ${_SECRET_NAME} \
```

Comment:
Great! This is a fail safe for if the user needs to update a sql config secret since it will error out if you try to recreate the secret. Though you can add the new version and reference "latest"